src/DonationForms/resources/app/DonationFormApp.tsx

```diff
@@ -36,28 +38,34 @@ const initialState = {
     validationSchema: schema,
 };
 
+const formSettings = {...form.settings, ...getDonationFormNodeSettings(form)};
```

Comment:
Missing `unreleased` tag here. Not sure about how are the criteria for it in JS/TS files.